### DISCUSSION TITLE
Prisma multiple schemas connections

### DISCUSSION THREAD

**Discussion by Dani-D99** [prisma/prisma#18797] [Q&A]
Hi,

I would like in my microservices project to be able to have different packages, for example v1, v2, v3, etc.
I would like to be able to have multiple connections in the prism and to be able to use microservices and the other database.
For example, to have a database: v1 and another database v2.
I read an article and saw that it is possible, but I don't know how to do it.
Can you give me a clear idea?

**Comment by nurul3101** [comment]
Hello @Dani-D99 👋 

Can you please have a look at this GitHub Issue:
- https://github.com/prisma/prisma/issues/2443

And in particular, this [comment](https://github.com/prisma/prisma/issues/2443#issuecomment-630679118) describes how you can have multiple PrismaClients for different schema files and different databases.

**Reply by nurul3101** [reply]
@Dani-D99 If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
@Dani-D99 I am checking to see if you had a chance to look at my last response.
